app/controllers/places_controller.rb

```diff
@@ -22,10 +22,10 @@ def create
       render :new
     end
   end
```

Comment:
You do still need a `def show; end` here so that Rails can take you from /places/3 to your erb, which launches you into React. This action just doesn't need to do any computations (before you were picking out the Place from the database)

Reply:
Got it, thanks for clarifying!